build.sbt

```diff
@@ -1,66 +1,63 @@
-import sbt.Keys.resolvers
 
-assemblyMergeStrategy in assembly := {
+name := "ingestion3"
+organization := "dpla"
+version := "0.0.1"
+scalaVersion := "2.11.8"
+
+parallelExecution in Test := false
+
+assembly / assemblyMergeStrategy := {
   case "META-INF/MANIFEST.MF" => MergeStrategy.discard
   case x => MergeStrategy.first
 }
 
-lazy val root = (project in file("."))
-  .configs(IntegrationTest)
-  .settings(
-    name := "ingestion3",
-    version := "0.0.1",
-    scalaVersion := "2.11.8",
-    Defaults.itSettings,
-    parallelExecution in Test := false,
-    libraryDependencies ++= Seq(
-      "org.scalatest" %% "scalatest" % "3.0.1" % "it,test",
+libraryDependencies ++= Seq(
+  // "org.scalatest" %% "scalatest" % "3.0.1" % "it,test",
```

Comment:
where is scalatest coming from? we should ban it from there and pin our version rather than accepting whatever a different project sticks us with

Reply:
i should have just removed `it` because the `it` package was removed from settings (not used afaik)

Reply:
i think mark had something in there to test the geocoding stuff at one point, but...